src/App.js

```diff
@@ -1,13 +1,50 @@
 import React from "react";
-
-import Bookings from "./Bookings";
+import Heading from "./Heading.js";
+import TouristInfoCards from "./TouristInfoCards.js";
+import Bookings from "./Bookings.js";
+import Footer from "./Footer.js";
+import SearchResults from "./SearchResults.js";
```

Comment:
Error warning: `SearchResults` is defined but never used.

I imagine this one was hard to do! Let's speak about it in our next catch up :)